tests/integration_tests/functional/test_rng.py

```diff
@@ -32,6 +32,21 @@ def list_rng_available(ssh_connection: SSHConnection) -> list[str]:
     )
 
 
+def get_rng_current(ssh_connection: SSHConnection) -> str:
+    """Returns the current rng device used by hwrng"""
+    return ssh_connection.check_output(
+        "cat /sys/class/misc/hw_random/rng_current"
+    ).stdout.strip()
```

Comment:
Tangential comment: It may be useful to have an "open this file in the Microvm" helper. Maybe sthg for the future.

Reply:
I thought about that but I didn't find other places in the code where it would be useful. We might add it in the future, but using `cat` seems "hacky".